src/components/StatusBadge/_StatusBadge.scss

```diff
@@ -4,8 +4,9 @@
   display: block;
   font-size: var(--font-size--1);
   font-style: italic;
-  padding: var(--space-xxs);
+  padding: var(--space-xxs) var(--space-xs);
   text-align: center;
+  width: fit-content;
```

Comment:
A width of `fit-content` might not work properly on Firefox unless you use a vendor prefix.  Could you please verify that this looks good on FF (I'd do it myself but I am still trying to get a license for BrowserStack )

CC @bigfishdesign13

Reply:
@wluisi Yeah, it looks like you'll probably need the browser prefix. Here's what it loox like on FF:
![image](https://user-images.githubusercontent.com/6432596/117673523-db65b600-b178-11eb-9b45-4a483424c8dc.png)

Reply:
That's funny--I had thought that we had an auto-prefixer as well.  I'll mention that to Marty and see if we can make that happen.